### ISSUE TITLE
Update WP-CLI

### ISSUE THREAD

**Issue by dealancer** [docksal/docksal#494]
With a new WP update, it requires a newer WP-CLI: 

```
      [W100] Your WP-CLI is too old; version 1.5.0 or newer is required. (RuntimeException)
```

It may result in failing of build scripts.

**Comment by dealancer** [comment]
I am checking with a newer version of containers.

**Comment by dealancer** [comment]
Tested with `web:2.0-apache2.4` and `cli:2.0-php7.2` there is the same problem.

also with this images there were another issue:

```
error sending mail
2018/04/05 17:38:23 dial tcp 141.161.99.99:1025: getsockopt: connection timed out
```

**Comment by dealancer** [comment]
Looks like in web:edge-apache2.4 and cli:edge-php7.2 this problem was resolved, so I am closing this issue and adding a new one re time out.

**Comment by lmakarov** [comment]
`wp-cli` v1.5.0 will be released in `docksal/cli` v2.1.0 - https://github.com/docksal/service-cli/pull/33
